### PR TITLE
refactor: lodash to native

### PR DESCRIPTION
**Pull request by curbengh**

### CODE REVIEW

**Comment by curbengh** [comment]
to be eslint-ed in a separate PR.

**Comment by curbengh** [comment]
rebased

File: lib/server.js [hunk 1]
```diff
@@ -11,14 +10,16 @@ module.exports = (args, callback) => {
...
+  let webuiConfig = Object.assign({}, {
     password: util.randomString(8, {uppercase: false}),
     port: 5000
   });
 
+  if (typeof config.webui !== 'undefined') {
+    webuiConfig = Object.assign(webuiConfig, config.webui);
+  }
+
   const app = express();
```
Comment (segayuu) [inline lib/server.js:23]: The first argument of `Object.assign()` is the object to which the property is assigned.
Therefore, there is no need to reassign this line to `webuiConfig`.
Therefore, `webuiConfig` can be declared by const statement.

Comment (curbengh) [reply lib/server.js:23]: Addressed. I also refactored to `if (config.webui)` since `config` is always defined.

File: api/controllers/files.js [hunk 2]
```diff
@@ -55,7 +54,10 @@ const readdir = (path, stats, hidden, callback) => {
     }, err => {
       if (err) return callback(err);
 
-      entry = _.sortBy(entry, 'name');
+      entry.sort((a, b) => {
+        return a.name.localeCompare(b.name);
+      });
+
       callback(null, data);
```
Comment (segayuu) [inline api/controllers/files.js:61]: I don't know if the result of `String#localeCompare()` and `lodash.sortBy()` are the same

Comment (curbengh) [reply api/controllers/files.js:61]: Good call. I also not sure if it's supposed to return the whole `entry` or just `entry.name`. I'll do more testing to confirm.

On the other hand, [localCompare()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#Sorting_non-ASCII_characters) has the advantage of supporting non-ASCII characters.

Comment (curbengh) [reply api/controllers/files.js:61]: Just tried
```js
const _ = require('lodash')

let entry = [
{ name: 'check', age: 45 },
{ name: 'base', age: 68 },
{ name: 'éservé', age: 75 },
{ name: 'eserve', age: 23 }
]

//entry = _.sortBy(entry, 'name');

entry.sort((a, b) => {
  return a.name.localeCompare(b.name);
});

console.log(entry)
```

Both yield 
```
[
  { name: 'base', age: 68 },
  { name: 'check', age: 45 },
  { name: 'eserve', age: 23 },
  { name: 'éservé', age: 75 }
]
```

Comment (segayuu) [reply api/controllers/files.js:61]: Thanks!

**Review by segayuu — APPROVED** [review]

**Pull request merged**